tests/unit/hosts/test_views.py

```diff
@@ -573,3 +573,135 @@ def test_when_requesting_user_is_not_part_of_team(self):
         response = self.client.post(self.url, self.data)
         self.assertEqual(response.data, expected)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+
+class DeleteChallengeHostFromTeamTest(BaseAPITestClass):
+    def setUp(self):
+        super(DeleteChallengeHostFromTeamTest, self).setUp()
+
+        self.challenge_host = ChallengeHost.objects.create(
+            user=self.user,
+            team_name=self.challenge_host_team,
+            status=ChallengeHost.ACCEPTED,
+            permissions=ChallengeHost.ADMIN,
+        )
+
+        self.user2 = User.objects.create(
+            username="user2",
+            email="user2@platform.com",
+            password="user2_password",
+        )
+
+        self.challenge_host2 = ChallengeHost.objects.create(
+            user=self.user2,
+            team_name=self.challenge_host_team,
+            status=ChallengeHost.ACCEPTED,
+            permissions=ChallengeHost.ACCEPTED,
+        )
+
+        self.url = reverse_lazy(
+            "hosts:delete_challenge_host_from_team",
+            kwargs={
+                "challenge_host_team_pk": self.challenge_host_team.pk,
+                "challenge_host_pk": self.invite_user.pk,
+            },
+        )
+
+    def test_challenge_host_does_not_exist_in_team(self):
+        self.url = reverse_lazy(
+            "hosts:delete_challenge_host_from_team",
+            kwargs={
+                "challenge_host_team_pk": self.challenge_host_team.pk,
+                "challenge_host_pk": self.challenge_host2.pk + 1,
+            },
+        )
+
+        expected = {"error": "ChallengeHost does not exist"}
+
+        response = self.client.delete(self.url, {})
+        self.assertEqual(response.data, expected)
+        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
+
+    def test_when_challenge_host_team_does_not_exist(self):
```

Comment:
Similarly, here. We are naming tests with format `test_perform_functionality_when_some_condition`. We could follow the same convention.